admin-auth/src/main/kotlin/com/epam/drill/admin/auth/config/ApiKeyAuthenticationProvider.kt

```diff
@@ -34,6 +36,10 @@ class ApiKeyAuthenticationProvider internal constructor(
     internal val authenticationFunction = config.authenticationFunction
     internal val challengeFunction = config.challengeFunction
     internal val authScheme = config.authScheme
+    internal val cache = Caffeine.newBuilder()
+        .maximumSize(config.maximumCacheSize)
+        .expireAfterWrite(Duration.ofDays(30))
```

Comment:
Why 30 days, why not get it from the config?

Reply:
1h by default like jwt expiration

Reply:
done